dashboard/common/db_util.py

```diff
@@ -12,16 +13,38 @@
 
 logger = logging.getLogger(__name__)
 
+def canvas_id_to_prefixed_id(canvas_id):
+    """
+    Insert the Canvas ID into the beginging of the DATA_WAREHOUSE_ID_TEMPLATE string
+    Ex: if DATA_WAREHOUSE_ID_TEMPLATE=11000000 and the Canvas ID = 999 then the prefixed_id will be 11000999
+    """
+    canvas_id = str(canvas_id)
+    prefixed_id = str(settings.DATA_WAREHOUSE_ID_TEMPLATE)[0:-len(canvas_id)] + canvas_id
+    return prefixed_id
+
+def prefixed_id_to_canvas_id(prefixed_id):
+    """
+    Remove the prefix from prefix_id to get the original Canvas ID
```

Comment:
I think we're actually going to have to change this up to treat this value as an integer rather than a string. This should simplify a lot of the code here.

Reply:
very true, I'll fix it up